src/styles/blocks/card.scss

```
@@ -0,0 +1,61 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  padding: 15px;
+  
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: flex;
+    align-self: center;
+    margin: 15px 4px 45px;
+  }
+  &__about {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 15px;
+    margin-top: 0;
+    margin-bottom: 6px;
+  }
+  &__code {
+    margin-bottom: 15px;
+  }
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 20px;
+  }
+  &__cost {
+    display: flex;
+    align-items: baseline;
+    justify-content: space-between;
+    margin-bottom: 20px;
+  }
+  &__amount {
+    font-size: 16px;
+    font-weight: 700;
+  }
+  &__button {
+    width: 170px;

```

```suggestion
    width: 100%;

```
Якщо зміниться ширина карточки, то в тебе кнопка не розтягнеться.